react/features/subtitles/components/web/Captions.tsx

```diff
@@ -1,25 +1,73 @@
+import { Theme } from '@mui/material';
 import React, { ReactElement } from 'react';
 import { connect } from 'react-redux';
+import { withStyles } from 'tss-react/mui';
 
 import { IReduxState } from '../../../app/types';
 import { getLocalParticipant } from '../../../base/participants/functions';
 import { getLargeVideoParticipant } from '../../../large-video/functions';
 import { isLayoutTileView } from '../../../video-layout/functions.web';
+import { calculateSubtitlesFontSize } from '../../functions.web';
 import {
     AbstractCaptions,
     type IAbstractCaptionsProps,
     _abstractMapStateToProps
 } from '../AbstractCaptions';
 
-
 interface IProps extends IAbstractCaptionsProps {
 
+    /**
+     * The height of the visible area.
+     */
+    _clientHeight?: number;
+
     /**
      * Whether the subtitles container is lifted above the invite box.
      */
     _isLifted: boolean | undefined;
+
+    /**
+     * An object containing the CSS classes.
+     */
+    classes?: Partial<Record<keyof ReturnType<typeof styles>, string>>;
 }
 
+
+const styles = (_theme: Theme, props: IProps) => {
+    const { _isLifted = false, _clientHeight } = props;
+
+    const toolbarSize = 48;
+    const fontSize = calculateSubtitlesFontSize(_clientHeight);
+
+    return {
+        transcriptionSubtitles: {
+            bottom: _isLifted ? `${toolbarSize + 36 + 40}px` : `${toolbarSize + 40}px`,
```

Comment:
Where do those 36 and 40 come from?

Reply:
ditto. 

For `36` I guess  the intention was to lift it for the label for the on stage participant's name. Will add a comment or something.

For `40` I can only guess this only the desired padding...

Reply:
Now I see that this whole thing is kind of dependent on the onstage participant display name badge, which uses the theme. I'll move those in a common place and use it from there.

Reply:
This should be fixed now. both stage participant display name badge and the captions are using the same thing from a generic function which uses the theme.